mx.truffleruby/native-image.properties

```diff
@@ -16,4 +16,5 @@ Args = -H:MaxRuntimeCompileMethods=7500 \
 # ${.} expands to the destination Ruby home created by mx fetch-languages,
 # such as substratevm/svmbuild/native-image-root/languages/ruby.
 JavaArgs = -Dtruffleruby.preinitialization.home=${.} \
-           -Dpolyglot.engine.PreinitializeContexts=ruby
+           -Dpolyglot.engine.PreinitializeContexts=ruby \
+           -Dorg.graalvm.launcher.standalone=true
```

Comment:
FWIW this will also means it's set in GraalVM afaik.

Reply:
yeah, I have the change locally, has to be done when this goes into GVM.

Reply:
It sounds simpler to just have a property only defined in GraalVM.

Reply:
yeah, but that requires more coordination and a proper solution, that's outside of scope of this simple fix. There is an issue created and assigned to Gilles, he'll look into this later.